### PR TITLE
Add MultiPEN wrappers

### PR DESCRIPTION
**Pull request by PerlaTroncosoRey**

### CODE REVIEW

File: tools/MultiPEN/MultiPEN-cross-validation.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <inputs>
+        <param name="expr_levels" type="data" format="tabular" label="Expression levels for features" help="Gene expression and/or metabolite levels" />
+        <param name="interaction_network" type="data" format="tabular" label="Interaction Network" help="Matrix with the interactions, each interaction is of the form: [node, node, interaction_weight]" />
+        <param name="classes" type="data" format="txt" label="Classes for the samples" help="1 for case or 0 for control" />
+        <param name="lambda" type="float" value="" label="Lambda for the optimisation function" help="" />
+        <param name="n_folds" type="integer" value="" label="N-folds for cross validation" help="" />
+        <param name="max_iter" type="integer" value="" label="Maximum number of iterations for optimisation" help="Optional parameter, by default this is set to 100" />
+    </inputs>
+    <outputs>
+        <data format="txt" name="statistics" label="${tool.name} on ${on_string} (lambda $lambda): cross-validation_statistics" from_work_dir="cross-validation_stats.txt" />
```
Comment (nsoranzo) [inline tools/MultiPEN/MultiPEN-cross-validation.xml:24]: Are you interested in having also the other 5 files that you are renaming in `<command>` as output datasets? They are missing here. If not, you can remove the `mv` commands.

File: tools/MultiPEN/MultiPEN-cross-validation.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    
+]]>
+    </command>
+    <inputs>
+        <param name="expr_levels" type="data" format="tabular" label="Expression levels for features" help="Gene expression and/or metabolite levels" />
+        <param name="interaction_network" type="data" format="tabular" label="Interaction Network" help="Matrix with the interactions, each interaction is of the form: [node, node, interaction_weight]" />
+        <param name="classes" type="data" format="txt" label="Classes for the samples" help="1 for case or 0 for control" />
+        <param name="lambda" type="float" value="" label="Lambda for the optimisation function" help="" />
+        <param name="n_folds" type="integer" value="" label="N-folds for cross validation" help="" />
+        <param name="max_iter" type="integer" value="" label="Maximum number of iterations for optimisation" help="Optional parameter, by default this is set to 100" />
```
Comment (nsoranzo) [inline tools/MultiPEN/MultiPEN-cross-validation.xml]: If this is optional, can you add `optional="true"` (usually we put it between `value` and `label`)?

File: tools/MultiPEN/MultiPEN-cross-validation.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    </inputs>
+    <outputs>
+        <data format="txt" name="statistics" label="${tool.name} on ${on_string} (lambda $lambda): cross-validation_statistics" from_work_dir="cross-validation_stats.txt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="expr_levels" ftype="tabular" value="X.txt" />
+            <param name="network_nodes" ftype="tabular" value="E.txt" />
```
Comment (nsoranzo) [inline tools/MultiPEN/MultiPEN-cross-validation.xml]: This should be `interaction_network` not `network_nodes`

File: tools/MultiPEN/MultiPEN-feature-selection.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        <data format="txt" name="Rankings" label="${tool.name} on ${on_string} (lambda $lambda): Rankings" from_work_dir="Rankings.txt" />
+        <data format="matlab" name="Rankings_mat" label="${tool.name} on ${on_string} (lambda $lambda): Rankings MATLAB" hidden="true" from_work_dir="Rankings.mat" />
+        <data format="txt" name="vts" label="${tool.name} on ${on_string} (lambda $lambda): vts" from_work_dir="vts.txt" />
+        <data format="txt" name="Rankings-higherInCases" label="${tool.name} on ${on_string} (lambda $lambda): Rankings-higherInCases" from_work_dir="Rankings-higherInCases.txt" />
+        <data format="txt" name="Rankings-higherInControl" label="${tool.name} on ${on_string} (lambda $lambda): Rankings-higherInControl" from_work_dir="Rankings-higherInControl.txt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="expr_levels" ftype="tabular" value="X.txt" />
+            <param name="network_nodes" ftype="tabular" value="E.txt" />
```
Comment (nsoranzo) [inline tools/MultiPEN/MultiPEN-feature-selection.xml]: This should be `interaction_network` not `network_nodes`

File: tools/MultiPEN/MultiPEN-feature-selection.xml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            <param name="classes" ftype="tabular" value="Y.txt" />
```
Comment (nsoranzo) [inline tools/MultiPEN/MultiPEN-feature-selection.xml]: `ftype` for `classes` should be `txt`

File: tools/MultiPEN/MultiPEN-cross-validation.xml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            <param name="classes" ftype="tabular" value="Y.txt" />
```
Comment (nsoranzo) [inline tools/MultiPEN/MultiPEN-cross-validation.xml]: `ftype` for `classes` should be `txt`

**Comment by nsoranzo** [comment]
@PerlaTroncosoRey I've added a couple of comments on the code, if you have time to fix them, otherwise I can do that after merging the pull request.

**Comment by PerlaTroncosoRey** [comment]
@nsoranzo working on the comments, good spotted!  I'll update wrappers before you do the merging. Thanks!

**Comment by nsoranzo** [comment]
Thanks @PerlaTroncosoRey , I'm going to delete the genepen wrapper, ok?

**Comment by PerlaTroncosoRey** [comment]
@nsoranzo ok about deleting genepen wrapper! thanks!

**Comment by nsoranzo** [comment]
Done!

**Pull request merged**